Makefile

```diff
@@ -60,7 +61,20 @@ $(ENV_DIR)/bin/python:
 include build-infra/cw-deb.mk
 
 .PHONY: build-eggs
-build-eggs: ${ENV_DIR}/.cluster-mgr-build-eggs ${ENV_DIR}/.config-mgr-build-eggs
+build-eggs: ${ENV_DIR}/.cluster-mgr-build-eggs ${ENV_DIR}/.queue-mgr-build-eggs ${ENV_DIR}/.config-mgr-build-eggs
+
+${ENV_DIR}/.queue-mgr-build-eggs: queue_mgr_setup.py shared_setup.py common/setup.py $(shell find src/metaswitch -type f -not -name "*.pyc") $(shell find common/metaswitch -type f -not -name "*.pyc") src/metaswitch/clearwater/queue_manager/alarm_constants.py
+        # Generate .egg files
+	${ENV_DIR}/bin/python queue_mgr_setup.py bdist_egg -d queue_mgr_eggs
```

Comment:
This should be `build -b build_queuemgr bdist_egg` - see https://github.com/Metaswitch/clearwater-etcd/pull/238/files.